PWGHF/Tasks/taskMcEfficiencyToXiPi.cxx

```diff
@@ -0,0 +1,435 @@
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskMcEfficiencyToXiPi.cxx
+/// \brief Task to analyse the MC efficiency for toXiPi charm baryon decay ("found / trackable")
+///
+/// \author Federica Zanone, Heidelberg University
+
+#include "CommonConstants/PhysicsConstants.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/O2DatabasePDGPlugin.h"
+#include "Framework/runDataProcessing.h"
+
+#include "Common/Core/RecoDecay.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+
+#include "PWGHF/Core/HfHelper.h"
+#include "PWGHF/DataModel/CandidateReconstructionTables.h"
+#include "PWGHF/DataModel/CandidateSelectionTables.h"
+
+using namespace o2;
+using namespace o2::analysis;
+using namespace o2::constants::physics;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct HfTaskMcEfficiencyToXiPi {
+
+  Configurable<double> rapidityCharmBaryonMax{"rapidityCharmBaryonMax", 0.5, "Max absolute value of rapidity for charm baryon"};
+  Configurable<double> mcLfAcceptanceEta{"mcLfAcceptanceEta", 1.0, "Max absolute value of eta for LF daughters"};
+  Configurable<double> mcPionFromCascadeAcceptancePt{"mcPionFromCascadeAcceptancePt", 0.2, "Min value of pt for pion <-- cascade"};
+  Configurable<double> mcPionFromCharmBaryonAcceptanceEta{"mcPionFromCharmBaryonAcceptanceEta", 0.8, "Max absolute value of eta for pion <-- charm baryon"};
+  Configurable<double> mcPionFromCharmBaryonAcceptancePt{"mcPionFromCharmBaryonAcceptancePt", 0.5, "Min value of pt for pion <-- charm baryon"};
+
+  Configurable<int> nClustersTpcMin{"nClustersTpcMin", 70, "Minimum number of TPC clusters requirement for pion <-- charm baryon"};
+  Configurable<int> nClustersItsMin{"nClustersItsMin", 3, "Minimum number of ITS clusters requirement for pion <- charm baryon"};
+
+  ConfigurableAxis axisPt{"axisPt", {200, 0, 20}, "pT axis"};
+  ConfigurableAxis axisMass{"axisMass", {900, 2.1, 3}, "m_inv axis"};
+
+  Service<o2::framework::O2DatabasePDG> pdg;
+  HfHelper hfHelper;
+
+  enum HFStep { kHFStepMC = 0,
+                kHFStepMcInRapidity,
+                kHFStepAcceptance,
+                kHFStepTrackable,
+                kHFStepAcceptanceTrackable,
+                kHFStepItsTpcTrackableCuts,
+                kHFStepItsTrackableCuts,
+                kHFStepTracked,
+                kHFStepTrackedCuts,
+                kHFStepTrackedSelected,
+                kHFNSteps };
+
+  enum TrackableStep { kTrackableAll = 0,
+                       kTrackableITS,
+                       kTrackableTPC,
+                       kTrackableITSTPC,
+                       kNTrackableSteps };
```

Comment:
Remove the `k` prefixes. Add documentation

Reply:
I would leave the k to be consistent with the stye of the other task for efficiency calculation, see https://github.com/AliceO2Group/O2Physics/blob/b2decb815c1cd6d434b428b74582297fba0192a7/PWGHF/Tasks/taskMcEfficiency.cxx#L51-L67

I added documentation.

Reply:
OK, then it was probably overlooked in the other task before. The conventions forbid the prefixes in general.